modules/appeals_api/app/swagger/notice_of_disagreements/v0/api_description_dev.md

```diff
@@ -28,18 +30,18 @@ To check the status of an appeal as it will appear to a claimant, use the Appeal
 | error   | An error occurred. See the error code and message for further information. |
 
 ### Status simulation in sandbox and staging
-Test submissions do not progress through the same statuses as the production environment. In sandbox and staging, the final status of a submission is `submitted`; however, we allow passing a `Status-Simulation` header on the show endpoints so that you can simulate all production statuses. 
+Test submissions do not progress through the same statuses as the production environment. In sandbox and staging, the final status of a submission is `submitted`; however, we allow passing a `Status-Simulation` header on the show endpoints so that you can simulate all production statuses.
 
 Statuses can also be simulated for evidence document uploads.
 
 ### Evidence uploads
 Our NOD evidence submission endpoints allow a client to upload a document package (documents and metadata) of supporting evidence for their submission by following these steps.
-1. Use the `POST /evidence_submissions` to return a JSON service response with the attributes listed below.
+1. Use the `POST endpoint /notice_of_disagreements/evidence_submissions` to return a JSON service response with the attributes listed below.
```

Comment:
I think that the following might be slightly easier to read here:
`POST /notice_of_disagreements/evidence_submissions` endpoint

Reply:
oops, this was a mistake - thanks!

Reply:
Fixed a few mistakes like these - in the process, I did a close read of these updates and found a few places where endpoints were incorrect (including here - I had included the old decision reviews route instead of the new segmented one)